model/account.rb

```diff
@@ -12,6 +12,10 @@ def hyper_tag_name(project = nil)
 
   include Authorization::TaggableMethods
 
+  def self.remove_all_active_sessions(id)
+    DB[:account_active_session_keys].where(account_id: id).delete(force: true)
```

Comment:
what would happen if this db operation fails? As in, is it possible that a user, say, resets password and this db operation fails hence the active sessions remain untouched? Is there any retry mechanism for the db operations?

Reply:
I guess we should check the transaction boundary with regard to these hooks.

Reply:
"Jeremy Evans thinks of everything" ![image](https://github.com/ubicloud/ubicloud/assets/7191/f5403b42-577f-48f3-a199-dcdd00350447)